src/app/submit-item/components/submit/submit.component.html

```diff
@@ -169,4 +169,16 @@ <h1 *ngIf="mailGiven">Nur noch ein Schritt - <span class="alt"> Mailadresse best
       <img height="380" src="assets/images/illustrations/undraw_with_love_ajy1.svg" width="380" />
     </div>
   </div>
-</ng-template>
\ No newline at end of file
+</ng-template>
+
+<ng-template #solved>
+  <div class="page-container-column">
+    <h1>Das kommt uns doch bekannt vor...</h1>
+    <p>Diesen Fall haben unsere Detektive schon bearbeitet und mit folgendem Vertrauensindex bewertet:</p>
+    <app-result-score [score]="item.result_score"></app-result-score>
+    <p>Weitere Details zu diesem Fall kannst du dir im Archiv ansehen.</p>
+    <button mat-raised-button color="accent" [routerLink]="['/archive', item.id]">Zum Archiv</button>
```

Comment:
This link does not work, because it links to `../archive/<ID>` instead of `../archive?id=<ID>`

Reply:
Done